s3prl/downstream/mos_prediction/README.md

```diff
@@ -0,0 +1,71 @@
+# MOS Prediction
+
+Official Implementation of "Utilizing Self-supervised Representations for MOS Prediction", will be presented at INTERSPEECH 2021 [[arXiv](https://arxiv.org/abs/2104.03017)]
+
+This code enables you to fine-tune a automatic Mean Opinion Score (MOS) Predictor with specific self-supervised upstream model.
+
+## Data Preparation
```

Comment:
I think it will be really nice if you add a section most interesting to most of the people before this line: **How to get use pretrained MOS models**. Instead of the example file (which is actually not required), it will be better to directly doc here for more visibility. I will image something like:

```
## Pretrained MOS models 

from s3prl.hub import wav2vec2_mos
model = wav2vec2_mos().cuda()
mos = model([wav1, wav2])
```